tests/crypto/mbedtls_psa/testcase.yaml

```diff
@@ -0,0 +1,37 @@
+# The goal here is to showcase that Mbed TLS's PSA implementation can be
+# supported on any platform. A very minimal configuration is tested here: simply
+# enable support for PSA APIs in Mbed TLS and then test them with:
+# - psa_crypto_init() which is required before any PSA operation
+# - psa_generate_random() which is always available as long as PSA is initialized.
+#
+# Since it might take too long to really execute this test on all Zephyr's
+# supported platforms, we reduce to a very small selected group whose
```

Comment:
```suggestion
# Since it might take too long to execute this test on all platforms
# supported by Zephyr, we reduce to a very small selected group whose
```